package.json

```diff
@@ -1,10 +1,11 @@
 {
   "name": "inscricao",
-  "version": "0.1.1",
+  "version": "1.0.0",
   "private": true,
   "dependencies": {
     "axios": "^0.21.2",
     "commitizen": "^4.2.4",
+    "dotenv": "^16.0.0",
```

Comment:
Ué, precisa? Não é só add `REACT_APP...`

Reply:
precisar n precisa se for add o prefix `REACT_APP` , faz mais sentido?